jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/Resource.java

```diff
@@ -1329,8 +1352,12 @@ public Collection<Resource> getAllResources()
      * <p>
      *     Each part of the input string could be path references (unix or windows style), or string URI references.
      * </p>
+     * <p>
+     *     If the result of processing the input segment is a java archive, then its resulting URI will be a mountable URI as `jar:file:...!/`.
+     * </p>
      *
      * @param str the input string of references
+     * @see #toJarFileUri(URI)
      */
     public static List<URI> split(String str)
```

Comment:
Since that's completely `Resource`-agnostic, this can probably be moved to `URIUtil`.

Reply:
Ditto, the future FileID/URIUtil PR.